test/unit/middleware/auth/sign_up_with_email_middleware_test.dart

```diff
@@ -18,12 +17,12 @@ void main() {
       // build state with email and password
       final testEmail = 'test@email.com';
       final testPassword = 'test_password';
-      final testAppState = AppState.init().rebuild((b) => b
-        ..emailAuthOptionsPage.email = testEmail
-        ..emailAuthOptionsPage.password = testPassword);
 
-      // build test store
-      final testStore = DispatchVerifyingStore(initialState: testAppState);
+      // Create a fake store and an action to push around.
```

Comment:
Effectively just different syntax for the same thing here, now using the FakeStore.